src/sentry/static/sentry/app/views/performance/trends/changedTransactions.tsx

```diff
@@ -0,0 +1,298 @@
+  transactions: NormalizedTrendsTransaction[];
+  location: Location;
+  index: number;
+  handleSelectTransaction: (transaction: NormalizedTrendsTransaction) => void;
+};
+
+function TrendsListItem(props: TrendsListItemProps) {
+  const {
+    transaction,
+    transactions,
+    trendChangeType,
+    index,
+    location,
+    handleSelectTransaction,
+  } = props;
+  const color = trendToColor[trendChangeType];
+
+  const selectedTransaction = getSelectedTransaction(
+    location,
+    trendChangeType,
+    transactions
+  );
+  const isSelected = selectedTransaction === transaction.transaction;
+
+  return (
+    <ListItemContainer>
+      <ItemRadioContainer color={color}>
+        <RadioLineItem index={index} role="radio">
+          <Radio
+            checked={isSelected}
+            onChange={() => handleSelectTransaction(transaction)}
+          />
+        </RadioLineItem>
+      </ItemRadioContainer>
+      <ItemTransactionNameContainer>
+        <ItemTransactionName>
+          <TransactionLink {...props} />
+        </ItemTransactionName>
+        <ItemTransactionAbsoluteFaster>
+          {transformDeltaSpread(
+            transaction.aggregate_range_1,
+            transaction.aggregate_range_2
+          )}
+        </ItemTransactionAbsoluteFaster>
+      </ItemTransactionNameContainer>
+      <ItemTransactionPercentContainer>
+        <Tooltip
+          title={
+            <TooltipContent>
+              <span>{transaction.project}</span>
+              <span>
+                <Count value={transaction.count_range_1} />
+                {' → '}
+                <Count value={transaction.count_range_2} />
+              </span>
+            </TooltipContent>
+          }
+        >
+          <ItemTransactionPercent>
+            {formatPercentage(transaction.divide_aggregate_range_2_aggregate_range_1, 0)}
+          </ItemTransactionPercent>
+        </Tooltip>
+        <ItemTransactionPercentFaster color={color}>
+          {transformDurationDelta(
+            transaction.minus_aggregate_range_2_aggregate_range_1,
+            trendChangeType
+          )}
+        </ItemTransactionPercentFaster>
+      </ItemTransactionPercentContainer>
+    </ListItemContainer>
+  );
+}
+
+type TransactionLinkProps = TrendsListItemProps & {};
+
+const TransactionLink = (props: TransactionLinkProps) => {
+  const {organization, trendView: eventView, transaction} = props;
+
+  const summaryView = eventView.clone();
+  const target = transactionSummaryRouteWithQuery({
+    orgSlug: organization.slug,
+    transaction: String(transaction.transaction),
+    query: summaryView.generateQueryStringObject(),
+  });
+
+  return <StyledLink to={target}>{transaction.transaction}</StyledLink>;
+};
+
+const StyledLink = styled(Link)`
+  word-break: break-all;
+`;
+
+const TransactionsList = styled('div')``;
+const ListItemContainer = styled('div')`
+  display: flex;
+  border-top: 1px solid ${p => p.theme.borderLight};
```

Comment:
something to be aware of here https://github.com/getsentry/sentry/pull/20363

Reply:
yeah I saw it, thanks.